code/controllers/subsystem/job.dm

```diff
@@ -926,6 +926,10 @@ SUBSYSTEM_DEF(job)
 		JobDebug("[debug_prefix] Error: [get_job_unavailable_error_message(JOB_UNAVAILABLE_FLAVOUR)], Player: [player][add_job_to_log ? ", Job: [possible_job]" : ""]")
 		return JOB_UNAVAILABLE_FLAVOUR
 
+	if((player.client.prefs.read_preference(/datum/preference/numeric/age) < MIN_COMMAND_AGE) && (possible_job.departments_bitflags & DEPARTMENT_BITFLAG_COMMAND))
```

Comment:
Seconding the first comment, there should be exemptions for Silicons as their age is a much more open-ended thing to work with.
Wouldnt be too hard to add the check on this line, skip if species is ipc or silicon; but the choice is up to maintainers in the end